test/common/testlib.py

```diff
@@ -731,21 +732,88 @@ def post_github(self, number, err):
         if not github.available:
             return False
 
+        timestamp = datetime.datetime.now().isoformat()
+
         # Lookup the link being logged to
         context = testinfra.DEFAULT_IMAGE
-        link = ""
```

Comment:
Now that this is a massive function, it should probably move to testinfra.py. A function that updates a known issue in github.